Upstream: unsure
Reason: the meson build skips the necessary objcopy step for this test when compiling with clang
--- a/gio/tests/resources.c	2019-09-05 18:56:48.000000000 +0200
+++ b/gio/tests/resources.c	2019-11-16 23:26:17.880065908 +0100
@@ -993,8 +993,10 @@ main (int   argc,
   g_test_add_func ("/resource/automatic", test_resource_automatic);
   /* This only uses automatic resources too, so it tests the constructors and destructors */
   g_test_add_func ("/resource/module", test_resource_module);
+#ifndef __clang__
   g_test_add_func ("/resource/binary-linked", test_resource_binary_linked);
 #endif
+#endif
   g_test_add_func ("/resource/uri/query-info", test_uri_query_info);
   g_test_add_func ("/resource/uri/file", test_uri_file);
   g_test_add_func ("/resource/64k", test_resource_64k);
